assets/js/atomic/blocks/product-elements/summary/index.ts

```diff
@@ -28,7 +30,4 @@ const blockConfig = {
 	save: Save,
 };
 
-registerBlockType( 'woocommerce/product-summary', {
-	...sharedConfig,
-	...blockConfig,
-} );
+registerBlockType( 'woocommerce/product-summary', { ...blockConfig } );
```

Comment:
We merge the `sharedConfig` to the `blockConfig` above, so we can use `blockConfig` directly without the spread here.

```suggestion
registerBlockType( 'woocommerce/product-summary', blockConfig );
```